api-challenge/src/test/java/nix/nhrin/apichallenge/service/DeIdentificationPatientRecordServiceTest.java

```diff
@@ -0,0 +1,40 @@
+package nix.nhrin.apichallenge.service;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.time.LocalDate;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest
+public class DeIdentificationPatientRecordServiceTest {
+
+    @Autowired
+    private DeIdentificationPatientRecordService service;
+
+    @Test
+    public void testCalculateAge() {
```

Comment:
Usually we call test methods using BDD approach: shouldReturn...., shouldReplace..., shouldThrow...,

Reply:
renamed